### PR TITLE
Add cleanprofs data

### PR DESCRIPTION
**Pull request by Dennis-Rosenbaum**
@gertperdZA I use this module a lot. What I missed was the option to see if [Cleanprofs](https://www.cleanprofs.nl/ophaaldagen/) will stop by to clean my containers. Therefore I have added this in your module. In order to add the cleanprofs data the user can add 'showCleanprofsData: true' to their config. Only then the cleanprofs data is retrieved and added.

Would you be so kind to check this out and merge if you are ok with the change.

Example:
image

### CODE REVIEW

File: MMM-Afvalwijzer.js [hunk 1]
```diff
@@ -92,6 +93,7 @@ Module.register('MMM-Afvalwijzer', {
             case 'PLASTICPLUS':
```
Comment (gertperdZA) [inline MMM-Afvalwijzer.js]: 'pbd' should rather be used

File: MMM-Afvalwijzer.js [hunk 2]
```diff
@@ -92,6 +93,7 @@ Module.register('MMM-Afvalwijzer', {
...
                 color = "#e96c29";
                 break;
+            case 'papier':
```
Comment (gertperdZA) [inline MMM-Afvalwijzer.js:96]: can be removed as it only uses 'papier and karton'

**Comment by gertperdZA** [comment]
Hi Dennis. Thanks for the addition to add the Cleanprofs. I also use their services :) so great addition to the module!  I reviewed the code with some comments. Do you mind fixing them? Once fixed then i'll merge with main. My apologies also for taking so long to get back, I was away on holiday.

File: MMM-Afvalwijzer.js (hunk 2 above)
Comment (Dennis-Rosenbaum) [reply MMM-Afvalwijzer.js:96]: @gertperdZA 
My data does show 'papier' as an option. The paper bins didn't turn up as blue, therefore I have added it.
image

File: MMM-Afvalwijzer.js (hunk 1 above)
Comment (Dennis-Rosenbaum) [reply MMM-Afvalwijzer.js]: @gertperdZA 
I have updated that part: https://github.com/gertperdZA/MMM-Afvalwijzer/pull/6/commits/a48f304c32b1e3fd7e259e4c9a85649d58a42572

**Comment by Dennis-Rosenbaum** [comment]
@gertperdZA I hope you had a great holiday! 
That means that you didn't have to put your bins outside haha.
Thank you for your compliments, it certainly comes in handy as I always forget when to leave them outside for cleaning.

I have resolved the issues, please let me know if you have further questions.

**Pull request merged**